workflows/Utility/modules/BEAUti/workflow/Snakefile

```diff
@@ -0,0 +1,57 @@
+"""Bayesian Evolutionary Analysis Utility (BEAUti)
+
+BEAUti is a graphical user-interface (GUI) application for generating BEAST XML files.
+
+Note: This is a graphical / interactive module that requires user interaction, so it is not suitable for running in a headless (non-interactive) environment.
+
+Params:
+    infile (str): The name of the input NEXUS (.nex) file.
+    outfile (str): The name of the output BEAST (.xml) file.
+"""
+configfile: "config/config.yaml"
+import os
+
+indir = config["input_namespace"]
+outdir = config["output_namespace"]
+params = config["params"]
+infile = params["infile"]
+outfile = params["outfile"]
+
+rule run_beauti:
+    input:
+        infile = f"results/{indir}/{infile}",
+    output:
+        outfile = f"results/{outdir}/{outfile}",
+    log:
+        "log/run_beauti.log"
+    benchmark:
+        "benchmark/run_beauti.txt"
+    params:
+        infile = infile,
+        outfile = outfile,
+        infolder = os.path.abspath(f"results/{indir}/"),
+        outfolder = os.path.abspath(f"results/{outdir}/")
+    conda:
+        "envs/beast.yml"
+    shell:
+        """
+        # Generate symbolic link to the input and output directories
```

Comment:
worth setting `set -e` here

Reply:
Snakemake [implicitly uses strict mode](https://snakemake.readthedocs.io/en/v7.32.3/project_info/faq.html) for shell execution already.